cdi/src/main/java/io/smallrye/config/inject/InjectionMessages.java

```diff
@@ -11,35 +10,29 @@
 import org.jboss.logging.annotations.MessageBundle;
 import org.jboss.logging.annotations.Param;
 import org.jboss.logging.annotations.Pos;
-import org.jboss.logging.annotations.Signature;
 
 @MessageBundle(projectCode = "SRCFG", length = 5)
 public interface InjectionMessages {
     InjectionMessages msg = Messages.getBundle(InjectionMessages.class);
 
-    @Signature(messageIndex = 1, value = { String.class, String.class })
-    @Message(id = 2000, value = "No Config Value exists for required property %s")
-    ConfigException noConfigValue(@Param @Pos(1) String key);
+    @Message(id = 2000, value = "Failed to Inject @ConfigProperty for key %s since the config property could not be found in any config source")
+    ConfigInjectionException noConfigValue(@Param @Pos(1) String key);
 
-    @Signature(messageIndex = 1, value = { String.class, String.class, Throwable.class })
-    @Message(id = 2001, value = "Failed to retrieve config for key %s")
-    ConfigException retrieveConfigFailure(@Cause IllegalArgumentException cause, @Param @Pos(1) String key);
+    @Message(id = 2001, value = "Failed to Inject @ConfigProperty for key %s %s")
+    ConfigInjectionException retrieveConfigFailure(@Param @Pos(1) String key, @Param @Pos(2) String cuase, @Cause Exception e);
```

Comment:
This could probably include the target type too, if that would be useful?
```
@Message(id = 2001, value = "Failed to Inject @ConfigProperty for key %s to type %s %s")
ConfigInjectionException retrieveConfigFailure(@Param @Pos(1) String key, @Param @Pos(2) String type , @Param @Pos(3) String cuase, @Cause Exception e);
```

Reply:
I'm not sure what you mean by "type" here. If mean the type of the injection point (e.g. are we injecting this property as a `String` or as a `boolean`) then I don't think that's necessary.

However, I think it would be useful to say where we were injecting into (i.e. the field or method parameter annotated with `@ConfigProperty`) because that's where the user will have to look if they're not sure of the purpose of the missing property.

Reply:
Yes, I meant the type of the injection point :)

Interesting suggestion about the location of the `InjectionPoint`. Is there a way to get a nice concise description of this? I know the InjectionPoint.toString() is a bit nasty and long.

Reply:
Try `InjectionPoint.getMember().getName()` and see how that looks like.